src/java.base/share/classes/java/lang/Class.java

```diff
@@ -92,6 +92,7 @@
 import sun.reflect.generics.scope.ClassScope;
 import sun.reflect.annotation.*;
 
+import static jdk.internal.util.ModifiedUtf.utfLen;
```

Comment:
Do not use static imports across package boundaries, it make the code harder to read and to know where that method came from.